vspace/src/main/webapp/resources/bootstrap/css/bootstrap.css

```diff
@@ -5,7 +5,7 @@
  */
 /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
 html {
-  font-family: sans-serif;
```

Comment:
try just adding
```
* {
 font-family...
}
```
that should the apply for everything and wouldn't need to be added individually to tags

Reply:
"wouldn't need to be added individually to tags"